RFS/src/test/java/com/rfs/tracing/TestContext.java

```diff
@@ -0,0 +1,31 @@
+package com.rfs.tracing;
+
+import lombok.Getter;
+import org.opensearch.migrations.tracing.BacktracingContextTracker;
+import org.opensearch.migrations.tracing.IContextTracker;
+import org.opensearch.migrations.tracing.InMemoryInstrumentationBundle;
+
+public class TestContext extends RootRfsContext {  @Getter
```

Comment:
There are a couple of unused methods here - lets either remove them or lets refactor to converge on a single TestContext factory.